standard/classes.md

```diff
@@ -4040,6 +4066,8 @@ The *type* of an indexer declaration specifies the element type of the indexer i
 
 > *Note:* As indexers are designed to be used in array element-like contexts, the term *element type* as defined for an array is also used with an indexer. *end note*
 
+The *formal_parameter_list* specifies the parameters of the indexer. The formal parameter list of an indexer corresponds to that of a method ([§15.6.2](classes.md#1562-method-parameters)), except that at least one parameter shall be specified, and that the `this`, `in`, `out`, and `ref` parameter modifiers are not permitted.
```

Comment:
>`in` [](http://example.com/codeflow?start=265&length=4)

Are `in` parameters allowed? `object this[in int index] { get; }`

Reply:
@cston – Yes, they are, thanks for spotting this. Would you like to enter a suggested fix (using the +/- page icon in the editing bar just above a comment)?

Now I wonder where else `in` has been added by mistake…

Reply:
updated in upcoming commit.